specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/examples/PaymentMethodAtBillingProfile_Remove.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
+    "api-version": "2021-10-01",
+    "billingAccountName": "00000000-0000-0000-0000-000000000032:00000000-0000-0000-0000-000000000099_2019-05-31",
+    "billingProfileName": "ABC1-A1CD-AB1-BP1",
+    "paymentMethodName": "ABCDABCDABC0"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/paymentMethodLinks/operationResults/ed2975a8-8f77-43bb-a717-ace54326b14b?api-version=2021-10-01",
+        "Retry-After": "60"
+      }
+    },
+    "200": {},
+    "204": {},
+    "409": {
+      "body": {
+        "code": "PaymentMethodNotEligibleForDetach",
```

Comment:
This should comply with ARM error schema. (top level "error" property is missing)

Reply:
changed